react/features/welcome/actionTypes.js

```diff
@@ -1,3 +1,17 @@
+// @flow
+
+/**
+ * Action to update the default page index of the {@code WelcomePageLists}
+ * component.
+ *
+ * {
+ *     type: UPDATE_WELCOME_PAGE_LIST_DEFAULT_PAGE,
+ *     pageIndex: number
+ * }
+ */
+export const UPDATE_WELCOME_PAGE_LIST_DEFAULT_PAGE
+    = Symbol('UPDATE_WELCOME_PAGE_LIST_DEFAULT_PAGE');
```

Comment:
Can you please adopt the name "set" rather than "update"?

Reply:
Updated, thanks!